CONTRIBUTING.md

```diff
@@ -143,23 +139,44 @@ The chosen tool for vendoring code in the project is [dep](https://github.com/go
 
 ### How are releases made?
 
-Releases are made by the project lead when deemed necessary. If you want to request a new release then mention this on your PR or Issue.
+Releases are made by the *Project Lead* on a regular basis and when deemed necessary. If you want to request a new release then mention this on your PR or Issue.
 
 Releases are cut with `git` tags and a successful Travis build results in new binary artifacts and Docker images being published to the Docker Hub and Quay.io. See the "Build" badge on each GitHub README file for more.
 
 Credentials for the `openfaas` and/or `functions` Docker Hub and quay.io accounts are coordinated by the project lead using.
```

Comment:
I know this isn't part of the change, but I found this reading through the document in full. The sentence seems to end unexpectedly.

> are coordinated by the project lead using.

using _what_? Or, am I just reading it wrong?

Reply:
will fix this

Reply:
I like the way the it is rewritten, current version could be feel a little "harsh" for new contributors.

Reply:
Thank you